src/pages/addItem.test.js

```diff
@@ -741,7 +737,7 @@ describe('<AddItem/>', () => {
                 screen.getByTestId('saving-exchange-loader')
             ).toBeInTheDocument()
         )
-    })
+    }, 7000)
```

Comment:
I got a timeout failure on this test, so increasing timeout. I don't think that it's related to the change, but 🤷‍♂️